config/install/core.entity_view_display.node.page.default.yml

```diff
@@ -143,6 +145,21 @@ third_party_settings:
                 entity: layout_builder.entity
             weight: 2
             additional: {  }
+          23050431-0772-44a7-9c25-3a8047a3c59e:
+            uuid: 23050431-0772-44a7-9c25-3a8047a3c59e
```

Comment:
should this uuid be here?

Reply:
Yeah that UUID is for the config type and needs to be there for layout builder. it's not the same as the "global" uuid for configs